app/forms/declaration/UNDangerousGoodsCode.scala

```diff
@@ -43,8 +43,8 @@ object UNDangerousGoodsCode extends DeclarationPage {
   private def form2Model: (String, Option[String]) => UNDangerousGoodsCode = {
     case (hasCode, codeValue) =>
       hasCode match {
-        case yes => UNDangerousGoodsCode(codeValue)
-        case no  => UNDangerousGoodsCode(None)
+        case `yes` => UNDangerousGoodsCode(codeValue)
```

Comment:
I think having full name of this would be more readable and less error prone ex.
```
AllowedUNDangerousGoodsCodeAnswers.yes
```